src/composer.json

```diff
@@ -16,10 +16,13 @@
         "stof/doctrine-extensions-bundle": "^1.3",
         "symfony/asset": "4.3.*",
         "symfony/console": "4.3.*",
+        "symfony/debug-pack": "^1.0",
         "symfony/flex": "^1.3.1",
         "symfony/form": "4.3.*",
         "symfony/framework-bundle": "4.3.*",
+        "symfony/maker-bundle": "^1.12",
```

Comment:
I don't use this myself, but could this be added to the dev packages? Since we only use it in development

Reply:
Yes, I use that a lot

Reply:
It's your pull request, you should add it to the development packages :stuck_out_tongue: (it's in this file)